File: python/array-concatenation.py
# https://leetcode.com/problems/concatenation-of-array/
# 66ms
# 16.86MB
class Solution:
    def getConcatenation(self, nums: List[int]) -> List[int]:
        arr_size = len(nums)
        result = []
        for i in range(2):
            for j in range(len(nums)):
                result.append(nums[j])
        return result

File: golang/array/array-concatenation.go
// https://leetcode.com/problems/concatenation-of-array/
// 9ms
// 6.58MB

package main

func main() {

}

func getConcatenation(nums []int) []int {
	var result []int
	for i := 0; i < 2; i++ {
		for j := 0; j < len(nums); j++ {
			result = append(result, nums[j])
		}
	}
	return result
}
